ember-cli-build.js

```diff
@@ -17,12 +17,14 @@ module.exports = function (defaults) {
     }
   })
 
-  /*
-    This build file specifes the options for the dummy test app of this
-    addon, located in `/tests/dummy`
-    This build file does *not* influence how the addon or the app using it
-    behave. You most likely want to be modifying `./index.js` or app's build file
-  */
+  if (app.env === 'test') {
```

Comment:
I'm not sure that's necessary here. If we *do* want that, I think it belongs in github.com/ciena-frost/ember-frost-test